Source/santactl/Commands/sync/testdata/sync_preflight_lockdown.json

```diff
@@ -1 +1 @@
-{"whitelist_regex": null, "client_mode": "LOCKDOWN", "blacklist_regex": null, "batch_size": 100}
\ No newline at end of file
+{"whitelist_regex": null, "client_mode": "LOCKDOWN", "blacklist_regex": null, "batch_size": 100}
```

Comment:
update these while we are at it: `whitelist_regex` and `blacklist_regex`

Reply:
I've left this file as-is to test that the existing keys aren't broken. The file `sync_preflight_basic.json` has the new keys in